bluepyentity/app/explorer.py

```diff
@@ -199,27 +200,42 @@ def leaves(adt, agg):
             self._refresh_all(url)
             return
 
-    def on_key(self, event: events.Key) -> None:
-        """Manage key pressed events."""
-        if event.character == "f":
-            self._follow_cmd = not self._follow_cmd
-            if self._follow_cmd:
-                self.display_hints()
-            else:
-                self.hide_hints()
+    BINDINGS = [
+        ("f", "follow", "Open a link"),
+        ("b", "back", "Back"),
+        ("q", "quit", "Quit"),
```

Comment:
for some reason, it displays the key in uppercase. I guess this is a textual issue.

Reply:
yeah :(